packages/core/src/actors/promise.ts

```diff
@@ -42,6 +54,12 @@ export function fromPromise<T>(
       }
     },
     start: (state, { self }) => {
+      // TODO: determine how to allow customizing this so that promises
+      // can be restarted if necessary
+      if (state.status !== 'active') {
+        return;
+      }
```

Comment:
In a way, that's achievable today with:
```ts
const fromRestartablePromise: typeof fromPromise = (...args) => {
  const behavior = fromPromise(...args)
  behavior.getPersistedState = behavior.getInitialState
  return behavior
}
```

I agree that we might want to evaluate making this configurable but that can be done post-release.